Add `The Followpoint: Dreamxiety, the osu! Artist Extraordinaire` news post (#12058)
ppy/osu-wiki: news/2024/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!.md

---
layout: post
title: "The Followpoint: Dreamxiety, the Picasso of osu!"
date: 2024-09-02 12:00:00 +0000
series: The Followpoint
---

The Followpoint has came back with another interview! This time it will be very artistic you could say.

![](/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/banner.jpg)

> What's this? A new interivew from The Followpoint? That's right, today we are back with [Dreamxiety](https://osu.ppy.sh/users/13103233). You may know her for creating artworks for the seasonal art contests, and you've probably seen them in the main menu whenever you load up osu!, if you even have seasonal backgrounds enabled... why do you guys have them disabled?

Reviewer: ```suggestion
What's this? A new interview from The Followpoint? That's right, today we are back with [Dreamxiety](https://osu.ppy.sh/users/13103233). You may know her for creating artworks for the seasonal art contests, and you've probably seen them in the main menu whenever you load up osu!, if you even have seasonal backgrounds enabled... why do you guys have them disabled?
```

I don't know, it's just something I did when I was a child and I never stopped since then. Many children like drawing but drop it as they grow up, but I didn't. I was maybe like 3 or 4 when I started and I just kept going.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Utiba.jpeg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

What is your drawing process like? Is there a certain workflow you follow when working on each project, or does each project have a different process?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/13103233" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Dreamxiety.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/13103233">Dreamxiety</a></p>

> If I'm looking for ideas, I'll browse through the internet to find inspirations. For osu contests specifically, I often just search for art that has the theme. It's a pretty good method since what comes at the top of the search are the things people generally find appealing, and what people have in mind when the name of the theme comes up.

Reviewer: ```suggestion
If I'm looking for ideas, I'll browse through the internet to find inspirations. For osu! contests specifically, I often just search for art that has the theme. It's a pretty good method since what comes at the top of the search are the things people generally find appealing, and what people have in mind when the name of the theme comes up.
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Utiba.jpeg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

Let's talk about osu!'s seasonal art contests, what made you interested back when you participated for the first time?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/13103233" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Dreamxiety.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/13103233">Dreamxiety</a></p>

I remember that I was opening the osu! homepage and I saw the announcement for [2019 Halloween Contest](https://osu.ppy.sh/community/contests/81). Previously, I had no idea that osu! even hosts that so it was a surprise and I was hyped. I joined not expecting to get in but surprisingly I did and I was really happy.

> That was also the first time I got my supporter tag, which I thought was cool. Back then I've never got to enjoy any sort of paid perks in a game because I was just a broke student and my parents aren't a fan of that kinda stuff lol.

Reviewer: ```suggestion
That was also the first time I got my osu!supporter tag, which I thought was cool. Back then I've never got to enjoy any sort of paid perks in a game because I was just a broke student and my parents aren't a fan of that kinda stuff lol.
```

That was my first time doing a collab project with other artists and designers. I don't know if I'm exactly good at communicating with other people so I was nervous, but it actually turned out to be fun. Shoutout to [nepperoni](https://osu.ppy.sh/users/7082924) and [saturn](https://osu.ppy.sh/users/22001295) for the awesome background art, and [Sakura006](https://osu.ppy.sh/users/10365024) for animating and carrying the whole project!

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/2iYtNVtL9ds" frameborder="0" allowfullscreen></iframe>
</div>

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Utiba.jpeg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

> Many spheres of the community have people making illustrated art, such as for tournaments, skinning, storyboarding, manga, art contests, music releases and merchandise. What would you like to see more of regarding art in osu!, and are there avenues you would personally be interested in delving into more?

Reviewer 2: ```suggestion
Many spheres of the community have people making illustrated art, such as for tournaments, skinning, storyboarding, manga, art contests, music releases, and merchandise. What would you like to see more of regarding art in osu!, and are there avenues you would personally be interested in delving into more?
```

It's a valid way to enjoy art either way, but I often see art that required such a crazy amount of skills place so low in the contest and I kinda wish people could appreciate them more.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Utiba.jpeg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>

To finish things up, what does the future hold for you? Do you have anything else that you'd like to mention?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/13103233" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Dreamxiety.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/13103233">Dreamxiety</a></p>

> I'm hoping I could do more stuff for the game, especially for osu! world cups. Osu is one of the only couple of games I actually stayed playing for a long time. Other games I usually just play maybe for a few months at most then I drop them, but osu actually got me to keep coming back. So I think it's nice to put the stuff I do out there for others to look at and enjoy.

Reviewer: ```suggestion
I'm hoping I could do more stuff for the game, especially for osu! world cups. osu! is one of the only couple of games I actually stayed playing for a long time. Other games I usually just play maybe for a few months at most then I drop them, but osu actually got me to keep coming back. So I think it's nice to put the stuff I do out there for others to look at and enjoy.
```

My final message would be 1) Play catch, and 2) I like KAITO thighs.

</div>

---

> Huge thanks to Dreamxiety for being apart of this interview, and I hope you readers enjoyed reading this interview! We very much appreciate the support!

Reviewer 2: ```suggestion
Huge thanks to Dreamxiety for being a part of this interview, and I hope you readers enjoyed reading this interview! We very much appreciate the support!
```
